sdk/core/System.ClientModel/api/System.ClientModel.net6.0.cs

```diff
@@ -102,8 +95,14 @@ public void Send(System.ClientModel.Primitives.PipelineMessage message) { }
     public partial class ClientPipelineOptions
     {
         public ClientPipelineOptions() { }
-        public System.ClientModel.Primitives.LoggingOptions LoggingOptions { get { throw null; } set { } }
-        public System.ClientModel.Primitives.PipelinePolicy? LoggingPolicy { get { throw null; } set { } }
+        public System.Collections.Generic.IList<string> AllowedHeaderNames { get { throw null; } }
```

Comment:
Why add these to pipeline options and not a logging options type?

Reply:
I believe this is the pattern in the email thread, is this not what you and Krzysztof had decided?

Reply:
No - IIRC the pattern from the email thread said policies would take pipeline options.  However, we should still group logging options on a type for usability/discoverability.  I believe I CC'ed you on the teams chat where I had suggested we look at making policy constructors take only what they use rather than pipeline options.  I'm still interested in pursuing the latter approach, to make change tracking easier.